### DISCUSSION TITLE
Uploading Bruker timsTOF output files

### DISCUSSION THREAD

**Discussion by deeptijk** [PRIDE-Archive/pride-resources#32] [General]
Zip every .d folders which are (raw + other files) individually and upload to PRIDE.

**Comment by ypriverol** [comment]
Great @deeptijk for the explanation. More details, for every timsTOF experiment you will end up as user to have multiple `.d` folders:
 
 - sample1.d 
 - sample2.d 
 
Each `.d` folder represent one MS run, raw file. However, rather than using one file per raw file, timsTOF instruments create a folder `.d` extension that included the capture MS but also other files with indexes etc. We recommended compressing each folder into an independent file like: 

- sample1.d.zip
- sample2.d.zip
